test/jasmine/tests/titles_test.js

```diff
@@ -45,6 +46,24 @@ describe('editable titles', function() {
         return promise;
     }
 
+    function editTitle(letter, attr, text) {
+        return new Promise(function(resolve) {
+            gd.once('plotly_relayout', function(eventData) {
+                expect(eventData[attr]).toEqual(text, [letter, attr, eventData]);
+                setTimeout(resolve, 10);
```

Comment:
@etpinard nice editing tests in [finance](https://github.com/plotly/plotly.js/pull/1349/files#diff-7deb9e75965ca15196b29a1f7a4a9210R971)! And good call to include a test like that here too. The only thing I did differently is this: put resolve in a short `setTimeout` instead of manually removing the node. Not sure why `makeEditable` uses a [duration-0 transition](https://github.com/plotly/plotly.js/blob/master/src/lib/svg_text_utils.js#L500) rather than just removing the element synchronously... but this seems to play well with it.